bundle/ios/GutenbergNativeTranslations.swift

```diff
@@ -187,15 +181,13 @@ private func dummy() {
     _ = NSLocalizedString("Move block right from position %1$s to position %2$s", comment: "translators: accessibility text. %1: current block position (number). %2: next block position (number)")
     _ = NSLocalizedString("Move block up", comment: "")
     _ = NSLocalizedString("Move block up from row %1$s to row %2$s", comment: "translators: accessibility text. %1: current block position (number). %2: next block position (number)")
-    _ = NSLocalizedString("Move blocks", comment: "")
     _ = NSLocalizedString("Move Image Backward", comment: "")
     _ = NSLocalizedString("Move Image Forward", comment: "")
     _ = NSLocalizedString("Move to bottom", comment: "")
     _ = NSLocalizedString("Move to top", comment: "")
     _ = NSLocalizedString("Navigate Up", comment: "")
     _ = NSLocalizedString("Navigates to custom color picker", comment: "")
     _ = NSLocalizedString("Navigates to customize the gradient", comment: "")
-    _ = NSLocalizedString("Navigates to select %s", comment: "translators: %s: Select control button label e.g. \"Button width\"")
```

Comment:
@hypest can you help me understand why these changes occurred? Thank you.

Reply:
Aha, not exactly sure myself. @fluiddot (or @ceyhun) maybe you have some idea?

Reply:
The localizations string files are updated (via the `genstrings` command) when running the `npm run bundle` command:
https://github.com/wordpress-mobile/gutenberg-mobile/blob/3c906d04e6b0acf0b652ccaf082928e773c6145c/package.json#L66

The reason why this string got removed is related to an issue when parsing the files that contain that string:
- [gutenberg/packages/components/src/font-size-picker/index.native.js](https://github.com/WordPress/gutenberg/blob/039d087a127d59b524e41bb2104b9bbfe05750f2/packages/components/src/font-size-picker/index.native.js#L77)
- [packages/components/src/mobile/bottom-sheet-select-control/index.native.js](https://github.com/WordPress/gutenberg/blob/039d087a127d59b524e41bb2104b9bbfe05750f2/packages/components/src/mobile/bottom-sheet-select-control/index.native.js#L63)

### `font-size-picker`
**Command to test parsing:** `npm run wp -- i18n make-pot gutenberg/packages/components/src/font-size-picker --ignore-domain --debug test.pot`
**Error:** `Could not parse file index.native.js: Unexpected: > (line 86, column 4) (0.199s)`
**Line that failed:** https://github.com/WordPress/gutenberg/blob/039d087a127d59b524e41bb2104b9bbfe05750f2/packages/components/src/font-size-picker/index.native.js#L86

### `bottom-sheet-select-control`
**Command to test parsing:** `npm run wp -- i18n make-pot gutenberg/packages/components/src/mobile/bottom-sheet-select-control --ignore-domain --debug test.pot`
**Error:** `Could not parse file index.native.js: Unexpected: > (line 72, column 4) (0.209s)`
**Line that failed:** https://github.com/WordPress/gutenberg/blob/039d087a127d59b524e41bb2104b9bbfe05750f2/packages/components/src/mobile/bottom-sheet-select-control/index.native.js#L72

As you can see on both cases failed due to the use of short syntax of the fragment. This parsing issue is already fixed ([reference](https://github.com/wp-cli/i18n-command/issues/281)) on the WP-CLI command we use for this but until a new version of WP-CLI is released we'll, unfortunately, keep experiencing it 😞 .

In this case, I'd recommend reverting the change since the string is required and should be translated.

Reply:
Appreciate the investigation and the suggestion @fluiddot, thanks a mil!

Tangential but, maybe we talked about this in the past, could we perhaps patch-package WP-CLI until the fix is released?

Reply:
> Tangential but, maybe we talked about this in the past, could we perhaps patch-package WP-CLI until the fix is released?

Good point, I thought that by now a new WP-CLI version would have been released but since it's taking longer than expected, probably this approach should be considered 👍 .

Reply:
Thank you @hypest and @fluiddot. These two files have been reverted and squashed into 961edcdbb2dd6b1549d97f8155f3464f2df85abf.

Reply:
Looks good Tanner! Marking this thread as resolved 👍